prism-languages.php

```diff
@@ -45,7 +45,6 @@ function mkaz_code_syntax_block_get_supported_languages() {
 		"toml" => "TOML",
```

Comment:
:warning: **Warning**: Array double arrow not aligned correctly; expected 7 space(s) between &quot;&quot;toml&quot;&quot; and double arrow, but found 1 (*WordPress.Arrays.MultipleStatementAlignment.DoubleArrowNotAligned*).